Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1160,3 +1163,26 @@ extension BrowserViewController: ReaderModeBarViewDelegate {
         }
     }
 }
+
+private class BrowserScreenshotHelper: ScreenshotHelper {
+    private weak var controller: BrowserViewController?
+
+    init(controller: BrowserViewController) {
+        self.controller = controller
+    }
+
+    func takeScreenshot(tab: Browser, aspectRatio: Float?, quality: Float) -> UIImage? {
```

Comment:
I would turn `aspectRatio` into ` CGFloat` because where you use this you actually convert to `CGFloat` anyway. Everything related to graphics in UIKit uses CGFloat, including point sizes, aspect ratios, positions, etc. (And CGFloat is actually a Float on 32 bit but a Double on 64 bit - but that is an implementation detail)